src/UCTNode.h

```diff
@@ -21,17 +21,20 @@
 
 #include "config.h"
 
-#include <tuple>
 #include <atomic>
 #include <limits>
+#include <memory>
+#include <tuple>
+#include <vector>
 
 #include "SMP.h"
 #include "GameState.h"
 #include "Network.h"
 
 class UCTNode {
 public:
-    using sortnode_t = std::tuple<float, int, float, UCTNode*>;
+    // Should this have a pointer?
+    using sortnode_t = std::tuple<float, int, float, std::unique_ptr<UCTNode>>;
```

Comment:
Could be removed, is not used anymore.

Reply:
Thanks for this (and the other comments), I do appreciate a 2nd pair of fresh eyes looking at the change :)